modules/machineset-manually-scaling.adoc

```diff
@@ -36,7 +36,27 @@ $ oc edit machineset <machineset> -n openshift-machine-api
 You can scale the MachineSet up or down. It takes several minutes for the new
 machines to be available.
 +
+
+= MachineSet delete policy
+
+MachineSet delete policy defines how priority is assigned to nodes to delete when downscaling a MachineSet. There are three supported options - Random, Newest and Oldest. It default to Random which means that random machines are picked and deleted when downscaling MachineSets. The delete policy can be set according to the use case by modifying the particular MachineSet:
+
+----
+
+spec:
+  deletePolicy: <delete policy>
+  replicas: <desired replica count>
+
+----
+
+Specific machines can also be priortized for deletion by adding the annotation `machine.openshift.io/cluster-api-delete-machine` to the machine of interest regardless of the delete policy.
+
 [IMPORTANT]
 ====
 By default, the {product-title} router pods are deployed on workers. Because the router is required to access some cluster resources, including the web console, do not scale the worker MachineSet to `0` unless you first relocate the router pods.
 ====
+
+[NOTE]
+====
+Custom/Pet MachineSets can be used for the use cases which needs running services on particular nodes and have a need to be ignored by the controller during downscaling the MachineSets. This will prevent the services from getting disrupted as the scaling down the worker MachineSets would not affect it.
```

Comment:
for use cases requiring that services run on specific nodes and that those services are ignored by the controller when the MachineSets are scaling down. This prevents service disruption.